server/routes/bookAVisit.test.ts

```diff
@@ -151,7 +151,7 @@ describe('GET /book-a-visit/select-visitors', () => {
       },
     }
 
-    prisonerVisitorsService.getVisitors.mockResolvedValue(returnData)
+    prisonerVisitorsService.getApprovedVisitors.mockResolvedValue(returnData)
```

Comment:
Renamed this to make it match what it actually does. Could've added an arg to the function but it's obvious what it does when the function is named like this.